hw5 - weatherApp/JS/index.js

```diff
@@ -0,0 +1,129 @@
+let submitFormBtn = document.getElementById('submitFormBtn');
+let cityInput  = document.getElementById('city');
+let countryInput = document.getElementById('country');
+let form = document.forms.cityCountryInputs;
+let block = document.getElementById('block');
+
+let locationHTML = document.getElementById('location');
+let dateHTML = document.getElementById('date');
+let temperatureNumber = document.getElementById('temperatureNumber');
+let weatherDescription = document.getElementById('weatherDescription');
+let temperature = document.getElementById('temperature');
+let humidity = document.getElementById('humidity');
+let pressure = document.getElementById('pressure');
+let cloudCover = document.getElementById('cloudCover');
+let uvIndex = document.getElementById('uvIndex');
+let visibility = document.getElementById('visibility');
+let windSpeed = document.getElementById('windSpeed');
+let windDirection = document.getElementById('windDirection');
+let windDegree = document.getElementById('windDegree');
+
+let temperatureLocationBlock = document.getElementById('temperatureLocationBlock');
+let weatherDescriptionBlock = document.getElementById('weatherDescriptionBlock');
+let weatherInfo = document.getElementById('weatherInfo');
+let welcomePhrase = document.getElementById('welcomePhrase');
+
+let error = document.getElementById('error');
+
+let host = 'http://api.weatherstack.com/current?access_key=b27bbe6126910fbe5e44ee53a22f3937&query=';
+
+function displayWeather() {
+    fetch(host + cityInput.value + ',' + countryInput.value)
+        .then(cleanInput(cityInput, countryInput))
+        .then(res => res.json())
+        .then(res => saveInfo(res))
+        .then(res => addInfoOnPage(res))
+        .then(showBlocks)
+        .catch(errorMessage)
+
+        return false;
+}
+
+submitFormBtn.addEventListener('click', displayWeather);
+form.addEventListener('submit', function(){event.preventDefault()});
```

Comment:
It's highly recommended not to use global `event`, it's been deprecated and exists only for compatibility.
Instead, use it from function param:
```suggestion
form.addEventListener('submit', function (event) {
  event.preventDefault();
});
```
You could also avoid separate event listeners and subscribe only form submit (which will hadnle both click and enter btn press):

```suggestion
form.addEventListener('submit', function (event) {
  event.preventDefault();
  displayWeather();
});
```